features/steps/step_selenium_filter_people_alphabet.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: UTF-8 -*-
+
+# @mark.steps
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+
+from behave import given, when, then
+
+
+@given('we are on Authorized Page')
+def step_impl(context):
+    context.authorized_page.navigate()
+
+
+@when('we click People')
+def step_impl(context):
+    context.authorized_page.switch_to_people()
+
+
+@then('we are on People Page')
+def step_impl(context):
+    context.people_page.navigate()
+
+
+@when('we choose a letter')
+def step_impl(context):
+    context.people_page.create_list_name()
```

Comment:
Does it belong to different step?
Creating list_name as a side effect of  "we choose a letter" may lead to bugs when you will try to reuse this step

Reply:
I separated two functions